### DISCUSSION TITLE
Favorites & Folders

### DISCUSSION THREAD

**Discussion by belg1o** [ente-io/ente#135] [Enhancements]
# Favorites & Folders

## Feature Details:

Ability to favorite/pin to top auth key
Ability to group auth keys by folders / tags
Ability to custom sort the auth list would be a very nice to have as well
Detailed Description:

/

## Impact on Existing Features, if any:
/

**Comment by OddHouseGames** [comment]
Just moved from 2FAS to Ente and loving the community so far but miss the folder functionality from 2FAS. Would love to be able to have favs, work, personal etc..

**Comment by ua741** [comment]
In the latest version, you can now add tags and pin specific entries.
